src/fty_sensor_gpio.cc

```diff
@@ -154,18 +156,30 @@ int main (int argc, char *argv [])
     string template_filename = string("/usr/share/fty-sensor-gpio/data/") + string("DCS001.tpl");
     FILE *template_file = fopen(template_filename.c_str(), "r");
     if (!template_file) {
-        template_filename = string("./data/") + string("DCS001.tpl");
+        template_filename = string("./selftest-ro/data/") + string("DCS001.tpl");
```

Comment:
I think `selftest-ro` went in here wrongly, should've been 5 lines below (e.g. look for `data` under CWD or under sources)?